src/sage/graphs/generic_graph.py

```diff
@@ -243,7 +243,8 @@
     :meth:`~GenericGraph.connected_components_sizes` | Return the sizes of the connected components as a list.
     :meth:`~GenericGraph.blocks_and_cut_vertices` | Compute the blocks and cut vertices of the graph.
     :meth:`~GenericGraph.blocks_and_cuts_tree` | Compute the blocks-and-cuts tree of the graph.
-    :meth:`~GenericGraph.is_cut_edge` | Return ``True`` if the input edge is a cut-edge or a bridge.
+    :meth:`~GenericGraph.is_cut_edge` | Check whether the input edge is a cut-edge or a bridge.
+    :meth:`~GenericGraph.is_edge_cut` | Check whether ``edges`` is an edge cut of ``G``.
     :meth:`~GenericGraph.is_cut_vertex` | Return ``True`` if the input vertex is a cut-vertex.
```

Comment:
The same comment as above.

Reply:
```
Check whether the input edges form an edge cut
```

Reply:
No plan to resolve?

Reply:
No, not yet.

Reply:
You did it in `src/sage/graphs/connectivity.pyx` but not in `src/sage/graphs/generic_graph.py`.